internal/backend/gdrive/gdrive_internal_test.go

```diff
@@ -0,0 +1,201 @@
+package gdrive
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	rtest "github.com/restic/restic/internal/test"
+	drive "google.golang.org/api/drive/v3"
+)
+
+func assertNil(t *testing.T, s string) {
```

Comment:
It feels to me the function name is wrong, it should be `assertEmptyString` or so.